### PR TITLE
feat: status page improvements

### PR DESCRIPTION
**Pull request by chasers**
- [x] adds regions to pings
- [x] uses node short name

## Future work
- dropdowns to select `from` and `to` regions
- show seconds from last ping
- color something orage > red depending on latency and maybe seconds from last ping
- dropdown for sort by

### CODE REVIEW

File: lib/realtime/helpers.ex [hunk 1]
```diff
@@ -103,6 +103,54 @@ defmodule Realtime.Helpers do
...
+      iex> node = :"pink@10.0.1.1"
+      iex> Realtime.Helpers.short_node_id_from_name(node)
+      "10.0.1.1"
+  """
+
+  @spec short_node_id_from_name(atom()) :: String.t()
+  def short_node_id_from_name(name) when is_atom(name) do
+    [_, ip] = name |> Atom.to_string() |> String.split("@", parts: 2)
+
+    {:ok, ip} = ip |> handle_nohost() |> String.to_charlist() |> :inet.parse_address()
```
Comment (abc3) [inline lib/realtime/helpers.ex:132]: But this only works for IP addresses. the node with hostname, for example, node@localhost will produce an error

**Review by abc3 — APPROVED** [review]

File: lib/realtime/helpers.ex (hunk 1 above)
Comment (chasers) [reply lib/realtime/helpers.ex]: Fixed!

**Pull request merged**